src/codelabs/dart-cheatsheet.md

```diff
@@ -53,41 +57,36 @@ For example, `stringify(2, 3)` should return `'2 3'`.
 String stringify(int x, int y) {
   TODO('Return a formatted string here');
 }
-{$ end main.dart $}
-{$ begin solution.dart $}
-String stringify(int x, int y) {
-  return '$x $y';
-}
-{$ end solution.dart $}
-{$ begin test.dart $}
 
-void main() {
-  try {
-    final str = stringify(2, 3); 
 
-    if (str == '2 3') {
-      _result(true);
-    } else if (str == '23') {
-      _result(false, ['Test failed. It looks like you forgot the space!']);
-    } else {
-      _result(false, ['That\'s not quite right. Keep trying!']);
-    }
-  } on UnimplementedError {
-    _result(false, ['Test failed. Did you implement the method?']);
-  } catch (e) {
-    _result(false, ['Tried calling stringify(2, 3), but received an exception: ${e.runtimeType}']);
-  }
+// Tests your solution (Don't edit!): 
+void main() {
+  assert(stringify(2, 3) == '2 3',
+      "Your stringify method returned '${stringify(2, 3)}' instead of '2 3'");
+  print('Success!');
 }
-{$ end test.dart $}
-{$ begin hint.txt $}
-Both x and y are simple values,
-and Dart's string interpolation will handle
-converting them to string representations.
-All you need to do is use the $ operator to
-reference them inside single quotes, with a space in between.
-{$ end hint.txt $}
+{$ end main.dart $}
```

Comment:
Can you get rid of these `begin` and `end main.dart` tags across the file? Support for them will also be going away and they're not necessary with just one Dart file anyway.

Reply:
Oh, sure! Didn't know